integration-cli/docker_api_stats_test.go

```diff
@@ -107,9 +107,22 @@ func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 	if runtime.GOOS == "windows" {
 		countParam = "-n" // Ping count parameter is -n on Windows
 	}
-	pingout, err := exec.Command("ping", contIP, countParam, strconv.Itoa(numPings)).Output()
-	pingouts := string(pingout[:])
+	pingout, err := exec.Command("ping", contIP, countParam, strconv.Itoa(numPings)).CombinedOutput()
+	if err != nil && runtime.GOOS != "windows" {
```

Comment:
If `runtime.GOOS == "windows"` on L107 should be kept, I think `runtime.GOOS == "linux"` is better than `runtime.GOOS != "windows" ` for L111 (related issues: e.g. support Solaris https://github.com/docker/docker/pull/22091)